doc/user_guide/eels.rst

```diff
@@ -148,42 +148,10 @@ HyperSpy has created the model and configured it automatically:
     >>> m.components
        # |            Attribute Name |            Component Name |            Component Type
     ---- | ------------------------- | ------------------------- | -------------------------
-       0 |                background |                background |                  PowerLaw
+       0 |                  PowerLaw |                  PowerLaw |                  PowerLaw
        1 |                       N_K |                       N_K |                EELSCLEdge
        2 |                       B_K |                       B_K |                EELSCLEdge
 
-.. warning::
-
-   Notice that the PowerLaw component has been automatically renamed to
-   "background". This behaviour is deprecated and will be removed in
-   HyperSpy 0.9. From them on this component will keep its original name,
-   "PowerLaw".
-
-
-Furthermore, the components are available in the user namespace
-
-.. code-block:: python
-
-    >>> N_K
-    <N_K (EELSCLEdge component)>
-    >>> B_K
-    <B_K (EELSCLEdge component)>
-    >>> background
-    <background (PowerLaw component)>
-
-.. warning::
-
-   This feature is deprecated and will be removed in HyperSpy 0.9. To access
-   the automatically created component use the following equivalent syntax:
-
-    .. code-block:: python
-
-        >>> m.components.N_K
```

Comment:
This and the following lines should be left, but with altered text around it - we do need an example how to access the components.

Furthermore, I'm not sure if we still inject variables named after element symbol - I think (hope!) not, so the text after this deleted chunk should be removed / altered as well